### DISCUSSION TITLE
ssh: connect to host dokku.me port 22: Connection timed out

### DISCUSSION THREAD

**Discussion by babarali539** [dokku/dokku#5578] [Q&A]
Getting this error while deploying my app. Can anyone help me resolve this issue? Do I need to register a domain and then add that into dokku app? I am running this
`git push dokku main`
and the it shows the following error in console
```
ssh: connect to host dokku.me port 22: Connection timed out
fatal: Could not read from remote repository.

Please make sure you have the correct access rights
and the repository exists.
```
Any help would be much appreciated. Thanks in advance.

**Comment by josegonzalez** [comment]
Did you setup dokku with the Vagrant VM or with a server you own somewhere? If the latter, the hostname `dokku.me` is a placeholder in the docs, and you need to replace that with your own server's hostname.
